docs/release_notes/7.0.2-1.md

```diff
@@ -0,0 +1,46 @@
+# Percona Server for MongoDB 7.0.2-1 (Release Candidate) (2023-09-28) 
+
+[Installation](../install/index.md){.md-button}
+
+We are pleased to announce the release of the new major version of Percona Server for MongoDB 7.0.2-1 – the drop in replacement for MongoDB Community Edition. Percona Server for MongoDB supports MongoDB community protocols and drivers and enhances it with the enterprise-grade features for free, enabling you to effectively meet the requirements of your organization in terms of high-availability, reliability and data security. 
+
+## Upgrade considerations
+
+Starting with version 7.0, journaling is enabled by default in Percona Server for MongoDB and the `storage.journal.enabled` configuration option has no effect. Before the upgrade, edit the configuration file and remove the `storage.journal.enabled` option to ensure successful database start after the upgrade. See the [upgrade guide](../install/upgrade-from-60.md) for exact procedure.
```

Comment:
From this text user might think that `storage.journal.enabled` will prevent norml server start which is not true.
In fact we implemented special handling for the `storage.journal.enabled` option. In PSMDB 7.0 this option will be ignored and special warning explaining the option is ignored will be logged. See PSMDB-1335.

Reply:
Why do we ask users to remove this option from config file before the upgrade if this option is ignored?

Reply:
Reworked